.travis.yml

```diff
@@ -3,31 +3,35 @@ language: python
 sudo: false
 
 matrix:
-    include:
-        - python: 2.7
-          env: TOXENV=py27
-        - python: 3.3
-          env: TOXENV=py34
-        - python: 3.4
-          env: TOXENV=py34
-        - python: 3.5
-          env: TOXENV=py35
-        - python: pypy
-          env: TOXENV=pypy
-        - python: 3.5
-          env: TOXENV=py2-cover,py3-cover,coverage
-        - python: 3.5
-          env: TOXENV=docs
-        - python: 2.7
-          env: TOXENV=py27-pyramid12
-        - python: 2.7
-          env: TOXENV=py27-pyramid13
-        - python: 2.7
-          env: TOXENV=py27-pyramid14
-        - python: 2.7
-          env: TOXENV=py27-pyramid15
-        - python: 2.7
-          env: TOXENV=py27-pyramid16
+  include:
+    - python: 2.7
+      env: TOXENV=py27
+    - python: 3.4
+      env: TOXENV=py34
+    - python: 3.5
+      env: TOXENV=py35
+    - python: 3.6
+      env: TOXENV=py36
+    - python: nightly
+      env: TOXENV=py37
+    - python: pypy
+      env: TOXENV=pypy
+    - python: 3.5
+      env: TOXENV=py2-cover,py3-cover,coverage
+    - python: 3.5
+      env: TOXENV=docs
+    - python: 2.7
+      env: TOXENV=py27-pyramid12
+    - python: 2.7
+      env: TOXENV=py27-pyramid13
+    - python: 2.7
+      env: TOXENV=py27-pyramid14
+    - python: 2.7
+      env: TOXENV=py27-pyramid15
+    - python: 2.7
+      env: TOXENV=py27-pyramid16
```

Comment:
Do we need to keep testing against older, no-longer-changing Pyramid versions?

Reply:
I would think that is no longer necessary.